pkg/services/dashboards/dashboard.go

```diff
@@ -29,7 +29,7 @@ type DashboardProvisioningService interface {
 	DeleteOrphanedProvisionedDashboards(ctx context.Context, cmd *models.DeleteOrphanedProvisionedDashboardsCommand) error
 }
 
-//go:generate mockery --name Store --structname FakeDashboardStore --output database --outpkg database --filename database_mock.go
+//go:generate mockery --name Store --structname FakeDashboardStore --inpackage --filename database_mock.go
```

Comment:
👀 The abstractions team will have to review this change, it’s not our service to move things around in.

Reply:
The change was to remove a dependency cycle